ada_2_fhir-r4/zibs2020/ada_project_drivers/_driverInclude.xsl

```diff
@@ -283,23 +283,39 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                 <xsl:for-each select="visueel_resultaat">
                     <xsl:call-template name="nl-core-TextResult-Media"/>
                 </xsl:for-each>
-                <!--<xsl:for-each select="verrichting">
-                    <xsl:apply-templates select="nf:ada-resolve-reference(verrichting)" mode="nl-core-Procedure">
-                        <xsl:with-param name="subject" select="$subject"/>
-                        <xsl:with-param name="report" select="ancestor::tekst_uitslag"/>
-                    </xsl:apply-templates>
-                </xsl:for-each>-->
             </xsl:when>
             <xsl:when test="$localName = 'toedieningsafspraak'">
                 <xsl:apply-templates select="$in" mode="nl-core-AdministrationAgreement">
                     <xsl:with-param name="subject" select="$subject"/>
                 </xsl:apply-templates>
             </xsl:when>
             <xsl:when test="$localName = 'verrichting'">
-                <xsl:apply-templates select="$in" mode="nl-core-Procedure">
-                    <xsl:with-param name="subject" select="$subject"/>
-                    <xsl:with-param name="report" select="if (ancestor::tekst_uitslag) then ancestor::tekst_uitslag else ()"/>
-                </xsl:apply-templates>
+                <xsl:choose>
+                    <xsl:when test="verrichting_start_datum/@value">
```

Comment:
Wat should happen if verrichting_start_datum/@value is not empty and equal to today? 
Perhaps this requires a broader discussion: the functions 'isFuture' and 'isPast' do not encompass 'today' (which makes sense). Should we make sure that 'today' is always part of the 'otherwise' logic or implement a new function 'isToday' instead? In for instance the logic w.r.t. status in EpisodeOfCare, a startDate equal to 'today' will result in an active status, via the 'otherwise' logic.

Reply:
Assume this will be a separate project/branch. Merging this one

Reply:
Didn't merge it after all, but this PR is messed up somehow. Will open a new PR